src/main/java/com/exasol/adapter/request/parser/TablesMetadataParser.java

```diff
@@ -109,12 +109,20 @@ private DataType getDataType(final JsonObject dataType) {
         case "GEOMETRY":
             type = getGeometryDataType(dataType);
             break;
+        case "HASHTYPE":
+            type = getHashtypeDataType(dataType);
+            break;
```

Comment:
Remove breaks and return type directly. Allows immutability.

Reply:
done